aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/UpdateHandler.java

```diff
@@ -64,20 +64,26 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         return ProgressEvent.progress(desiredResourceState, callbackContext)
                 .then(progress -> {
                     if (shouldRemoveFromGlobalCluster(request.getPreviousResourceState(), request.getDesiredResourceState())) {
-                        return removeFromGlobalCluster(proxy, proxyClient, progress, request.getPreviousResourceState().getGlobalClusterIdentifier());
+                        return removeFromGlobalCluster(proxy, rdsProxyClient, progress, request.getPreviousResourceState().getGlobalClusterIdentifier());
+                    }
+                    return progress;
+                })
+                .then(progress -> {
+                    if (shouldSetDefaultVpcSGId(progress)) {
+                        return setDefaultVpcSecurityGroupIds(proxy, rdsProxyClient, ec2ProxyClient, progress);
```

Comment:
Nit: please align these method names. E.g.: `setDefaultVpcSecurityGroupIds/4` and `shouldSetDefaultVpcSecurityGroupIds/1`

Reply:
Done.